modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -91,7 +81,7 @@ To protect the rest of the cluster in the event of member loss, you can set a mi
 When configuring a Quorum Size for your cluster, you need to catch the thrown exception to make some sort of decision (e.g. send an email, stop process, logging, retry strategies, etc)
 ====
 
-To enable quorum, the +mule.cluster.quorumsize+ property has to be placed on the cluster configurations file +{MULE_HOME}/.mule/mule-cluster.properties+ defining the minimum number of nodes of the cluster to remain in an operational state.
+To enable quorum, the `+mule.cluster.quorumsize+` property has to be placed on the cluster configurations file `+{MULE_HOME}/.mule/mule-cluster.properties+` defining the minimum number of nodes of the cluster to remain in an operational state.
```

Comment:
```suggestion
To enable quorum, place in the cluster configurations file `+{MULE_HOME}/.mule/mule-cluster.properties+` the `+mule.cluster.quorumsize+` property, which defines the minimum number of nodes of the cluster to remain in an operational state.
```
I edited this assuming that the property is what defines, but did not at first know if it was the property, or the file, or the user (you). Please reread to ensure accuracy.